### PR TITLE
Add test for cmd/kubeadm/app/images/images.go

### PR DESCRIPTION
**Pull request by songxiao-wang87**
What type of PR is this?
/kind cleanup

What this PR does / why we need it:
add UT

Which issue(s) this PR fixes:
none

Special notes for your reviewer:
none

Does this PR introduce a user-facing change?
none

```release-note

NONE

```

### CODE REVIEW

**Review by yangjunmyfm192085 — COMMENTED** [review]
/ok-to-test
/lgtm

File: cmd/kubeadm/app/images/images_test.go [hunk 1]
```diff
@@ -262,6 +270,18 @@ func TestGetDNSImage(t *testing.T) {
...
 			},
 		},
+		{
+			expected: "foo.io/coredns/coredns:v1.10.0",
+			cfg: &kubeadmapi.ClusterConfiguration{
+				ImageRepository: "foo.io/coredns",
+				DNS: kubeadmapi.DNS{
+					Type: kubeadmapi.CoreDNS,
+					ImageMeta: kubeadmapi.ImageMeta{
+						ImageTag: "v1.10.0",
```
Comment (neolit123) [inline cmd/kubeadm/app/images/images_test.go:280]: what does this test case add that we are not testing already?

File: cmd/kubeadm/app/images/images_test.go [hunk 2]
```diff
@@ -75,6 +75,14 @@ func TestGetKubernetesImage(t *testing.T) {
 				KubernetesVersion: testversion,
 			},
 		},
+		{
+			image:    constants.KubeProxy,
+			expected: GetGenericImage(gcrPrefix, "kube-proxy", expected),
+			cfg: &kubeadmapi.ClusterConfiguration{
+				ImageRepository:   gcrPrefix,
+				KubernetesVersion: testversion,
+			},
```
Comment (neolit123) [inline cmd/kubeadm/app/images/images_test.go:84]: this addition seems ok, given kube proxy is also versioned the same way.

File: cmd/kubeadm/app/images/images_test.go (hunk 1 above)
Comment (songxiao-wang87) [reply cmd/kubeadm/app/images/images_test.go:280]: Yes, this test case tests the validity of ImageTag.

Comment (pacoxu) [reply cmd/kubeadm/app/images/images_test.go:280]: https://github.com/kubernetes/kubernetes/blob/e218e0ede4eef31027adb6c868b3b6099b25f02b/cmd/kubeadm/app/images/images.go#L59-L61

Line 60 is not covered before.

Comment (neolit123) [reply cmd/kubeadm/app/images/images_test.go:280]: you can just add it to the previous test and expect a custom tag?

Comment (songxiao-wang87) [reply cmd/kubeadm/app/images/images_test.go]: Thanks for your advice. I'll modify it.

**Review by neolit123 — COMMENTED** [review]
/lgtm

**Review by SataQiu — COMMENTED** [review]
thanks @songxiao-wang87 
/lgtm
/approve

**Pull request merged**